tests/test_runs/test_run_functions.py

```diff
@@ -87,10 +90,22 @@ def _check_serialized_optimized_run(self, run_id):
 
         return True
 
+    def _perform_run(self, task_id, num_instances, clf,
+                     random_state_value=None, check_setup=True):
+
+        def _remove_random_state(flow):
```

Comment:
It's removed after checking the value to make sure that `assert_flow_equal` works.